src/java.base/share/classes/java/lang/Double.java

```diff
@@ -968,19 +1001,29 @@ public static long doubleToLongBits(double value) {
     public static native double longBitsToDouble(long bits);
 
     /**
-     * Compares two {@code Double} objects numerically.  There
-     * are two ways in which comparisons performed by this method
-     * differ from those performed by the Java language numerical
-     * comparison operators ({@code <, <=, ==, >=, >})
-     * when applied to primitive {@code double} values:
-     * <ul><li>
-     *          {@code Double.NaN} is considered by this method
-     *          to be equal to itself and greater than all other
-     *          {@code double} values (including
-     *          {@code Double.POSITIVE_INFINITY}).
-     * <li>
-     *          {@code 0.0d} is considered by this method to be greater
-     *          than {@code -0.0d}.
+     * Compares two {@code Double} objects numerically.
+
+     * This method imposes a total order on {@code Double} objects
+     * with two differences compared to the incomplete order defined the
+     * by Java language numerical comparison operators ({@code <, <=,
```

Comment:
Typo: defined the by Java -> defined by the Java.

Reply:
Will fix; thanks.